crates/symbolicator-js/src/utils.rs

```diff
@@ -111,7 +111,9 @@ pub fn fixup_webpack_filename(filename: &str) -> String {
 
 pub fn is_in_app(abs_path: &str, filename: &str) -> Option<bool> {
     if abs_path.starts_with("webpack:") {
-        Some(filename.starts_with("./") && !filename.contains("/node_modules/"))
+        // This diverges from the original logic. Previously we would only consider
+        // a filename starting with `./` as in-app, but that seems to be overly strict.
+        Some(!filename.starts_with("~/") && !filename.contains("/node_modules/"))
```

Comment:
Can you please add a comment with a link to the original code?
It would help navigate back the Git history.

Reply:
Yes, good idea.